modules/scf/defaults.sh

```diff
@@ -13,6 +13,7 @@ OPERATOR_CHART_URL="${OPERATOR_CHART_URL:-latest}"
 
 # scf-gen-config relevant:
 
+SCF_SERVICES="${SCF_SERVICES:-lb}" # lb, ingress
```

Comment:
I was under impression that lb is used even for ingress. You can do lb without ingress but not ingress without lb. I might be wrong.

Reply:
AFAIU Ingress depends on the k8s provider, so it can be that you get an ingress termination without a LB.

:thinking:  maybe `SCF_SERVICES` is a misnomer, but I can't find anything better. It's an option to specify how the services communicate externally. There's 3 possibilities: nodeport (ip, as we did, but not useful for kubecf), lb (k8s provider LBs, what kubecf defaults to), ingress (k8s provider dependent, on GKE it actually spawns a GKE LB).